pages/main.js

```diff
@@ -1,4 +1,5 @@
 const $ = x => document.getElementById(x)
+
 var codepage = "λƛ¬∧⟑∨⟇÷×«␤»°•ß†€"
```

Comment:
We might want to consider adding a method inside the Scala.js code to get the codepage so it won't have to be duplicated here

Reply:
Oh yeah we can do that now. Very cool